lib/api/bulkActivityExtract.js

```diff
@@ -1,8 +1,8 @@
-var _ = require('lodash'),
+var https = require('https')
+  _ = require('lodash'),
   Promise = require('bluebird'),
   util = require('../util'),
   Retry = require('../retry'),
-  MarketoStream = require('../stream'),

```

Comment:
I guess we could remove this now?

Reply:
It is exported with here https://github.com/MadKudu/node-marketo/blob/e7e42a0f5069b1377ccce5fc5817806d9f852360/lib/marketo.js#L42-L44
It can be used by other clients.

I think we should keep it, until we do a major release

Reply:
Well, we could do one, no? Because we're totally changing the bulk streaming function.

Reply:
Yes

Reply:
As we saw earlier this will remove a feature for the users (stream the pagination-based fetches)